spec/mailers/previews/report_mailer_preview.rb

```diff
@@ -14,6 +14,24 @@ def monthly_key_metrics_report
       subject: 'Example Key Metrics Report',
       message: 'Key Metrics Report February 2021',
       tables: [
+        [
+          { title: 'February 2021 Active Users' },
+          ['Monthly Active Users', 'Value'],
+          ['IAL1', 1],
+          ['IDV', 1],
+          ['Total', 2],
+        ],
+        [
+          { title: 'Total user count (all-time)' },
+          ['All-time user count'],
+          [289411],
```

Comment:
```suggestion
          [2289411],
```

This is all sample data so it's not a big deal, but the sample data below shows 1,972,368 proofed identities, so this number should be greater than that. Some part of my brain otherwise gets thrown off trying to interpret it, even though it's not real.

Don't hold up the PR for this, though.